src/TobiasOlry/TalklyBundle/Security/NtlmAuthenticator.php

```diff
@@ -51,6 +58,10 @@ public function start(Request $request, AuthenticationException $authException =
      */
     public function getCredentials(Request $request)
     {
+        if (!$this->enable) {
```

Comment:
I would prefer 'passive' here, enabled over enable - since the authenticator gets enabled/disabled and does not enable itself.